.gitignore

```diff
@@ -0,0 +1,2 @@
+_site
+.*
```

Comment:
Really? isn't that just for files that start with .? so `markdown.md.swp` wouldn't be covered by that? If so the `.*` would ignore all files :P